pkg/signup/signup_service_test.go

```diff
@@ -86,8 +88,40 @@ func (s *TestSignupServiceSuite) TestFailsIfUserSignupNameAlreadyExists() {
 	require.EqualError(s.T(), err, fmt.Sprintf("usersignups.toolchain.dev.openshift.com \"%s\" already exists", userID.String()))
 }
 
+func (s *TestSignupServiceSuite) TestFailsIfUserBanned() {
```

Comment:
Can we also add a noise (not relevant banned user) to some existing signup service tests to make sure we don't fail if there is a banned user but with different hash.

Reply:
I've added a new test to cover this.